server/src/main.py

```diff
@@ -145,6 +147,17 @@ async def fastapi_lifespan(app: FastAPI):
 fastapi_app = WATcloudFastAPI(logger=logger, lifespan=fastapi_lifespan)
 transaction_lock = Lock()
 
+def resign_whitelist():
```

Comment:
This looks good, can we move this down beside housekeeping? Might also be a good idea to add `@app.command()` and `@fastapi_app.post` decorators so that we can manually trigger it via cli/REST.